linera-service/src/linera/client_context.rs

```diff
@@ -405,6 +407,54 @@ impl ClientContext {
             linera_service::node_service::wait_for_next_round(stream, timeout).await;
         }
     }
+
+    pub async fn change_ownership<S>(
+        &mut self,
+        chain_id: Option<ChainId>,
+        ownership_config: ChainOwnershipConfig,
+        force: bool,
+        storage: S,
+    ) -> anyhow::Result<()>
+    where
+        S: Storage + Clone + Send + Sync + 'static,
+        ViewError: From<S::ContextError>,
+    {
+        let chain_id = chain_id.unwrap_or_else(|| self.default_chain());
+        let chain_client = self.make_chain_client(storage, chain_id);
+        info!("Changing ownership for chain {}", chain_id);
+        let time_start = Instant::now();
+        let ownership = ChainOwnership::try_from(ownership_config)?;
+        anyhow::ensure!(
+            force || ownership.super_owners.is_empty(),
+            "Regular ownership is recommended to guarantee the chain's liveness. \
+            If you really want to set super owners, please use -f or --force."
+        );
```

Comment:
I'm not fond of this phrasing because this casts doubt on the super-owner mode without explaining the tradeoffs.